File: 2639-separate-the-digits-in-an-array/separate-the-digits-in-an-array.py
class Solution:
    def separateDigits(self, nums: List[int]) -> List[int]:
        ans=[]
        for i in range(len(nums)):
            a=list(str(nums[i]))
            for j in range(len(a)):
                a[j]=int(a[j])
                ans.append(a[j])
        return ans    

File: 2639-separate-the-digits-in-an-array/separate-the-digits-in-an-array.cpp
class Solution {
public:
    vector<int> separateDigits(vector<int>& nums) {
        vector<int>ans;
        for(int i=nums.size()-1;i>=0;i--){
            while(nums[i]>0){
                ans.push_back(nums[i]%10);
                nums[i]=nums[i]/10;
            }
        }
        reverse(ans.begin(),ans.end());
        return ans;
    }
};